LocalPayment/src/test/java/com/braintreepayments/api/localpayment/LocalPaymentNonceUnitTest.java

```diff
@@ -60,4 +61,32 @@ public void parcelsCorrectly() throws JSONException {
         assertEquals("9KQSUZTL7YZQ4", parceled.getPayerId());
         assertEquals("084afbf1db15445587d30bc120a23b09", parceled.getClientMetadataId());
     }
+
+    @Test
+    public void when_fromJSON_is_called_missing_fields_default_to_an_empty_string() throws JSONException {
+        LocalPaymentNonce result = LocalPaymentNonce.fromJSON(
+            new JSONObject(Fixtures.PAYMENT_METHODS_LOCAL_PAYMENT_MISSING_FIELDS_RESPONSE)
+        );
+
+        Parcel parcel = Parcel.obtain();
+        result.writeToParcel(parcel, 0);
+        parcel.setDataPosition(0);
+
+        LocalPaymentNonce parceled = LocalPaymentNonce.CREATOR.createFromParcel(parcel);
```

Comment:
🧶: we can remove this portion of the test since we no longer own the `Parcelable` implementation.

Reply:
I have made changes for this file, will push those changes soon.

Reply:
Good call. I've removed the parcel logic from this test.

@ashivashankars - it should make things easier when merging this fix into your branch